ui/static/ui/js/learning_resources.jsx

```diff
@@ -135,12 +137,27 @@ define('learning_resources', [
         thiz.setState({
           message: "Form saved successfully!"
         });
+        thiz.hideLoaderAndScrollUp();
       }).fail(function () {
         thiz.setState({
           message: {error: "Unable to save form"}
         });
+        thiz.hideLoaderAndScrollUp();
       });
     },
+    hideLoaderAndScrollUp: function() {
+      var scrollableDiv = $(this.getDOMNode()).closest(
+        ".cd-panel .cd-panel-container .cd-panel-content"
+      );
+      var messageBox = $(this.getDOMNode()).find('.status-container');
+      $(this.getDOMNode()).find('.lr-spinner').css('display', 'none');
+      if (typeof messageBox !== 'undefined' && $(messageBox).text()) {
+        scrollableDiv.animate(
+          {scrollTop: messageBox.offset().top},
```

Comment:
You shouldn't need to scroll to the messageBox exactly, why not just scroll all the way to the top?